testUtils.ts

```diff
@@ -26,11 +27,32 @@ export const NuxtLinkStub = Vue.component<
   },
 });
 
+export const mockI18n = (options?: MockI18nOptions) => {
+  const mockEnglishLocale = { code: 'en', name: 'English' };
+
+  return {
+    $i18n: {
+      locales: [
+        mockEnglishLocale,
+        { code: 'es', name: 'Spanish' },
+        { code: 'fr', name: 'French' },
+        ...(options?.locales ?? []),
```

Comment:
So this isn't really overriding the locales, so much as mocking them out for testing purposes. If you look at the `LanguageSelector.test.ts` file, you can see that I'm using the result of this mock to setup the context for the component through the `@nuxtjs/composition-api`'s `useContext` hook, since that's how this is expected to be accessed. So rather than try to configure the `@nuxtjs/i18n` module to work within the tests (which I'm honestly not sure is possible or not), we're just giving it the dummy data it needs to test that the `<LanguageSelector />` functions as we would expect.

Realistically, the `<LanguageSelector />` component is the only one that I see using this mock, within the tests, but I figured that rather than set it up just for that one, I'd do it in a more available place, in case we need this elsewhere in the future.

Reply:
What I really mean here is that in all the calls to this function (e.g. in `LanguageSelector.test.ts`), you don't actually pass in an `options.locales` value. In fact, I don't see us ever calling `mockI18n()` with `locales`.